src/test/resources/org/jenkinsci/plugins/workflow/flow/FlowExecutionListTest/stepExecutionIteratorDoesNotLeakBuildsWhenOneIsStuck/jobs/test0/builds/1/build.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version='1.1' encoding='UTF-8'?>
+<flow-build plugin="workflow-job@1400.v7fd111b_ec82f">
+  <queueId>1</queueId>
+  <timestamp>1722979974600</timestamp>
+  <startTime>1722979974615</startTime>
+  <duration>0</duration>
+  <charset>UTF-8</charset>
+  <keepLog>false</keepLog>
+  <execution class="org.jenkinsci.plugins.workflow.cps.CpsFlowExecution">
+    <result>SUCCESS</result>
+    <script>stage(&apos;stage&apos;) { sleep 30 }; timeout(time: 10) { sleep 30 }</script>
+    <loadedScripts class="linked-hash-map"/>
+    <durabilityHint>MAX_SURVIVABILITY</durabilityHint>
+    <timings class="map">
+      <entry>
+        <string>flowNode</string>
+        <long>101709541</long>
+      </entry>
+      <entry>
+        <string>classLoad</string>
+        <long>124446251</long>
+      </entry>
+      <entry>
+        <string>run</string>
+        <long>200459289</long>
+      </entry>
+      <entry>
+        <string>parse</string>
+        <long>166818958</long>
+      </entry>
+      <entry>
+        <string>saveProgram</string>
+        <long>57936125</long>
+      </entry>
+    </timings>
+    <internalCalls class="sorted-set"/>
+    <sandbox>true</sandbox>
+    <iota>5</iota>
+    <head>1:5</head>
```

Comment:
These are notably older than the latest nodes in `workflow/`. In the real build that I saw, there was also a partially-written `atomic<numbers>.tmp` file for the build, indicative of some kind of uncontrolled Jenkins shutdown.